### PR TITLE
LPS-163350

### PR DESCRIPTION
**Pull request by ces-totrinh**
Hi @hongvo2308, please help review this PR and leave your comment. Thanks.
Ticket:
https://issues.liferay.com/browse/LPS-163350

### CODE REVIEW

File: modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/servlet/taglib/util/BlogsEntryActionDropdownItemsProvider.java [hunk 1]
```diff
@@ -314,6 +316,12 @@ private String _getPermissionsURL(BlogsEntry blogsEntry) {
 	}
 
 	private String _getRedirectURL() {
+		String redirect = ParamUtil.getString(_httpServletRequest, "redirect");
+
+		if (Validator.isNotNull(redirect)) {
```
Comment (ces-hongvo) [inline modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/servlet/taglib/util/BlogsEntryActionDropdownItemsProvider.java]: @tototrinh 
ParamUtil will return BLANK as default. So I think you should use !Validator.isBlank(...)

File: modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/view_entry_content_detail.jsp [hunk 2]
```diff
@@ -248,7 +249,13 @@ BlogsPortletInstanceConfiguration blogsPortletInstanceConfiguration = BlogsPortl
 							<liferay-asset:asset-tags-summary
 								className="<%= BlogsEntry.class.getName() %>"
 								classPK="<%= entry.getEntryId() %>"
-								portletURL="<%= renderResponse.createRenderURL() %>"
+								portletURL="<%=
```
Comment (ces-hongvo) [inline modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/view_entry_content_detail.jsp]: @tototrinh 
Please double check if the redirect will be BLANK as default and if this function work properly with the BLANK redirect.

**Review by hongvo2308 — CHANGES_REQUESTED** [review]
@tototrinh Please be noted my comments. And let me know if you have any concerns.
Thanks

File: modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/view_entry_content_detail.jsp (hunk 2 above)
Comment (ces-totrinh) [reply modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/view_entry_content_detail.jsp]: I have checked and see that the redirect couldn't be blank and the function is ok with the blank redirect. Thanks.

**Comment by ces-totrinh** [comment]
@hongvo2308, I updated the code please help take a look. Thanks.

**Comment by ces-hongvo** [comment]
Forwarded https://github.com/liferay-lima/liferay-portal/pull/3561.
Thank you.

**Pull request closed without merging**